modules/serverless-create-default-channel-yaml.adoc

```diff
@@ -6,12 +6,12 @@
 [id="serverless-create-default-channel-yaml_{context}"]
 = Creating a default implementation channel by using YAML
 
-You can create a channel with the cluster default configuration by using YAML.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe channels declaratively and in a reproducible manner. To create a serverless channel by using YAML, you must create a YAML file that defines a `Channel` object, then apply it by using the `oc apply` command.
```

Comment:
Not quite sure how to fix it, but _files uses_ seems awkward. 

_Creating Knative resources by using YAML files enables you_
_When you create Knative resources by using YAML files, you use a declarative API_ ??

Reply:
Would the second sentence be a stronger lead? 
_ You can create a serverless channel by using YAML by creating a YAML file that defines a `Channel` object, then apply it by using the `oc apply` command. reating Knative resources by using YAML files..._